examples/avro-cli.py

```diff
@@ -113,16 +129,19 @@ def produce(topic, conf):
     producer.flush()
 
 
-def consume(topic, conf):
+def consume(topic, conf, avro_serializer):
```

Comment:
con-fu-sing.  Should be avro_deserializer for sure. I can see it now.
+1 on split classory.